integration.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Compiling the webserver
+echo "Building webserver."
+make
+
+# starting webserver
+{	
+	echo "Starting webserver."
+	timeout 10 ./webserver test_config_1
+	echo "Stopping webserver."
+}&
+sleep 3
+
+# testing webserver
+echo "Running command curl -D headers.txt http://localhost:12345 > response.txt"
```

Comment:
probably not necesary now but for future assignments it might be better to dynamically create the config file or at least read the port from the config file into the script for ease of use, extendability and portability